### PR TITLE
met client requirement for hello world console printing

### PR DESCRIPTION
**Pull request by matts-dev**
Please do a code review so that we may add the feature to the application.

### CODE REVIEW

File: HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class HelloWorld {
+    public static void main(String[] args){
+	System.out.println("Hello World!");
```
Comment (fenwickjb) [inline HelloWorld.java]: Printed string does not meet specification.

**Review by fenwickjb — CHANGES_REQUESTED** [review]

**Comment by fenwickjb** [comment]
Would be better if testing were automated and not relying on humans who are error-prone. ;-)
Check this out...
http://stackoverflow.com/questions/1119385/junit-test-for-system-out-println

Also, it might help if you assign me as a "reviewer" (see panel to the right).

**Comment by matts-dev** [comment]
@fenwickjb Changed the class to better represent string and added a new unit test. 

*do we need to create a new pull request if changes are made to the commit?*

File: TestHelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import org.junit.Test;
+public class TestHelloWorld{
+	
+	@Test
+	public void testHelloWorldString(){
+		//save the system.in and system.out
```
Comment (fenwickjb) [inline TestHelloWorld.java:11]: Lines 11-17 are "before the test" setup types of activities. JUnit provides a nice mechanism for these setup and teardown types of things.

**Review by fenwickjb — CHANGES_REQUESTED** [review]

**Pull request merged**